vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Exhibition.java

```diff
@@ -48,15 +48,15 @@ public class Exhibition extends VSpaceElement implements IExhibition {
 
     private String customMessage;
     
-    @OneToOne(targetEntity = VSImage.class)
+    @OneToOne(targetEntity = VSImage.class, cascade = CascadeType.ALL, orphanRemoval=true)
     @NotFound(action = NotFoundAction.IGNORE)
     private IVSImage spaceLinkDefaultImage;
     
-    @OneToOne(targetEntity = VSImage.class)
+    @OneToOne(targetEntity = VSImage.class, cascade = CascadeType.ALL, orphanRemoval=true)
     @NotFound(action = NotFoundAction.IGNORE)
     private IVSImage moduleLinkDefaultImage;
     
-    @OneToOne(targetEntity = VSImage.class)
+    @OneToOne(targetEntity = VSImage.class, cascade = CascadeType.ALL, orphanRemoval=true)
```

Comment:
I don't think we want these properties. If a user decides to replace a default image, it should still be available to choose again later.